plugins/codesnippet/plugin.js

```diff
@@ -14,7 +14,7 @@
 
 	CKEDITOR.plugins.add( 'codesnippet', {
 		requires: 'widget,dialog',
-		lang: 'en', // %REMOVE_LINE_CORE%
+		lang: (window.__codesnippet__ && window.__codesnippet__.lang) || 'en', // %REMOVE_LINE_CORE%
```

Comment:
instead of exposing in `window` can we pass a param `codesnippetLang` in config from consumers ?

Reply:
I have to expose it using `window` because the somehow right config is not passed to this plugin from editor during initialization.   I have debugged a lot to figure this out but to no avail.